test/functional/rpc_psbt.py

```diff
@@ -230,6 +230,47 @@ def test_sighash_mismatch(self):
 
         wallet.unloadwallet()
 
+    def test_sighash_adding(self):
```

Comment:
Let's add a TODO in this file to write a test that checks `PSBT_GLOBAL_TX_MODIFIABLE` field is correctly set if `SIGHASH_SINGLE` is used in any of the inputs once the PSBT V2 PR https://github.com/bitcoin/bitcoin/pull/21283 is merged?

Reply:
I don't think it makes sense to do that here since PSBTv2 is still entirely unparseable.

Reply:
I know it's not parseable right now, that's why suggested only a TODO that'd be done after PSBTv2 gets merged. The assumption I have is this PR will get merged before #21283. I didn't notice any such test in the PSBT PR, which is fine because that PR is already big enough, and adding a TODO here can be a reminder for later.